core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/StackViewV4Response.java

```diff
@@ -176,4 +179,12 @@ public String getStackVersion() {
     public void setStackVersion(String stackVersion) {
         this.stackVersion = stackVersion;
     }
+
+    public String getVariant() {
```

Comment:
I know this is not a fault of your change, but perhaps it would make sense to add a `toString()` impl as well like for `StackV4Response`.